source/habitat_quality.rst

```diff
@@ -179,51 +179,54 @@ This section outlines the specific data used by the model. Please consult the In
 
 * **Results suffix** (optional). Text string that will be appended to the end of output file names, as "_Suffix". Use a Suffix to differentiate model runs, for example by providing a short name for each scenario. If a Suffix is not provided, or is not changed between model runs, the tool will overwrite previous results.
 
-* **Current Land Cover** (required). A GIS raster dataset, with an integer LULC code for each cell. The LULC raster should include the area of interest, as well as a buffer of the width of the greatest maximum threat distance. Otherwise, locations near the edge of the area of interest may have inflated habitat quality scores, because threats outside the area of interested are not properly accounted for. *The LULC codes must match the codes in the "Sensitivity of land cover types to each threat" table below.*
+* **Current Land Cover** (required). A GIS raster dataset, with an integer LULC code for each cell. The LULC raster should include the area of interest for the study, such that the area of interest is inset from the edges of the LULC raster by the width of the greatest maximum threat distance. Otherwise, locations near the edge of the area of interest may have inflated habitat quality scores, because threats outside the area of interest are not properly accounted for. This occurs when threats are at the edge of the LULC raster and the study area of interest is the entire LULC boundary. *The LULC codes must match the codes in the "Sensitivity of land cover types to each threat" table below.*
```

Comment:
I find both the old and new versions of this a little bit hard to parse, in terms of figuring out how I need to prepare my input data. So here's another attempt in which I try to avoid loaded terms like "buffer" and "area of interest". How does this read, and is it accurate?

"Habitat quality (_or whatever else_) values near the edges of the output rasters may be inflated because they do not account for threats that may exist beyond the extent of the land cover rasters. All input threat data are clipped to the extent of the LULC raster (_Is this true? I'm seeing this as the cause of the issue_). So, users should restrict interpretation of the results by disregarding habitat quality values that are within the `maximum threat distance` of the edge of the output rasters."

Is there any other option for users to avoid this issue? Like making sure their threat data extend beyond the bounds of the LULC (but that wouldn't work if the threats are clipped to the LULC up front).

Reply:
If I'm interpreting this issue correctly, maybe it's more an "Interpreting Results" paragraph rather than a "Data Needs" paragraph? Of course it could be linked and cross-referenced to both.

Reply:
Yea it's tricky, the only thing I can really think of is if we require users to provide threat rasters that extend beyond the LULC by `maximum threat distance` and then internally buffer the LULC with 0's or some value, so that operations would work properly. But then checking for and validating that would be cumbersome and handling the case where it's not true would put us back to where we are now. 

I like the way you've written about the issue here and I agree that it might be best to make it less of a _data need_ and more of a _interpreting results_ or _gotchya_ thing. For *Current LULC* I'm leaning towards mentioning that this raster will be used for defining the geospatial extents for corresponding threat rasters. Then leaving everything else in another section.